cme.thor

```diff
@@ -16,6 +16,10 @@ class CmeThor < Thor
       sleep 30 while group.alive?
       puts "Celluloid::Supervision::Container #{self} crashed. Restarting..."
     end
+  rescue SignalException => e
```

Comment:
why not https://github.com/honeybadger-io/honeybadger-ruby `exceptions.ignore` if all we want is to not honeybadger?